libsrc/common/stpcpy.c

```diff
@@ -0,0 +1,8 @@
+#include <string.h>
+
+char * __fastcall__
+stpcpy (char * dst, const char * src)
+{
```

Comment:
this should adhere to the codestyle of the rest of the library, ie use
`char* __fastcall__ stpcpy (char* dest, const char* src)`